group_vars/sn05.yml

```diff
@@ -70,11 +70,9 @@ postgresql_conf:
   - checkpoint_completion_target: 0.9
   - random_page_cost: 1.0
   - effective_cache_size: "16GB"
-# log dirs not set up yet
-#  - log_directory: "'/export/backups/pgsql/pg_log'"
-#  - log_checkpoints: "on"
 #  - log_line_prefix: "'%t:%r:%u@%d:[%p]<%m>: '"
-#  - log_min_duration_statement: 100
+  - log_checkpoints: "on"
+  - log_min_duration_statement: 100
```

Comment:
I think we had this once enabled, but it was creating a hug logfile. Please have a look at the grows if we enable that :)

Reply:
It was ok as long as the threshold was at 100 ms; when s/o lowered it to 10 ms the size of the log files "exploded". I'll keep an eye on it, but I think a threshold of 100 ms should be ok.